.github/workflows/_test.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     timeout-minutes: ${{ inputs.timeout-minutes }}
     env:
       DOCKER_IMAGE: ${{ inputs.docker-image }}
-      WORKDIR: /var/lib/jenkins/workspace
+      WORKDIR: /home
```

Comment:
the home dir is set to `/root`, so something like `/root/workspace` ? Likewise, you are picking up the htmlcov from `/root/htmlcov`.

Reply:
makes sense, updated to `/root/workspace`